blocks/identity-block/components/bot-challenge-protection/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { Paragraph, useIdentity } from "@wpmedia/arc-themes-components";
+import useSales from "../../utils/useSales";
+import ReCAPTCHA from "react-google-recaptcha";
+
+const BotChallengeProtection = ({ challengeIn, setCaptchaToken, className }) => {
+	const { Identity, isInitialized } = useIdentity();
+  const { Sales } = useSales();
+	const [siteKey, setSiteKey] = useState();
+	console.log('siteKey', siteKey)
+
+	const onChange = (value) => {
+		setCaptchaToken(value);
+    localStorage.setItem('captchaToken', captchaToken);
+	};
+
+	useEffect(() => {
+		const checkCaptcha = async () => {
+			const config = await Identity.getConfig();
```

Comment:
@edwardcho1231 I think you can move L23-L24 into `if` on L27, thus you call config from identity or sales, depending on challengeIn

Reply:
@LauraPinilla Don't we have to call `Identity.config` even when the `challengeIn === 'checkout'` since we need the recaptchaSiteKey?

Reply:
good point @edwardcho1231 I missed that. Yep for checkout we need to call both